main.js

```diff
@@ -0,0 +1,109 @@
+/*eslint-disable*/
+
+//gameboard object
+let gameBoard = {
+
+  // The pegs and there respective disks
+ pegs:[
+  [5,4,3,2,1],
+  [],
+  []
+ ],
+
+ 
+ //method to move around disks
+ move: function(chooseIndex, chooseDestination){
+
+
+    // this asks is the array you are choosing an ampty array. If it is you cant use it.
+    if(this.pegs[chooseIndex].length === 0){
+       console.log("Please choose a populated array")
+    }
+
+    else {
+      //This maps the pegs board 
+      let array = this.pegs.map(function(elem){
```

Comment:
The variable `array` is too generic here. I'm not sure why you needed to do this (by looking at the variable or reading the comment).